### DISCUSSION TITLE
Priority navigation component

### DISCUSSION THREAD

**Discussion by 7iomka** [mantinedev/mantine#2530] [Feature requests]
Currently for vertical (mobile) menus I use Navlink, which is awesome.
For toggable menus (like list of actions) I prefer to use Menu component.
However, when I have the horizontal menu, which has some container bounded width, but an unknown number of items, I have a problem with this case.
My suggestion is to create a component that would allow custom element lists to be manipulated so that those that don't fit to be fully readable would be placed in a separate component - this is can be a trigger (ActionIcon or just configurable ReactElement) with Menu component inside a popover or without,
This technique is also known as `priority navigation`.

An example of implementation in vanilla js: [DEMO](https://github.com/gijsroge/priority-navigation)
[Alternatives](https://github.com/gijsroge/priority-navigation#alternatives)

**Comment by rtivital** [comment]
Yes, it will be nice to have
